src/Raven.Studio/typescript/models/resources/info/databaseInfo.ts

```diff
@@ -44,6 +44,34 @@ class databaseInfo extends resourceInfo {
         this.documentsCount(databaseInfo.DocumentsCount);
         this.indexesCount(databaseInfo.IndexesCount);
     }
+
+    static empty(resourceName: string): databaseInfo {
```

Comment:
not necessarily it's true.  tal implemented cache so we can fetch info even about offline databases, but event had name loaded not load  ( changed few days ago by Danielle) so right now we have assurance that database is already loaded when this notification arrives. so we remove this empty method and load info and then display on screen